src/defaultsettings.cpp

```diff
@@ -429,7 +429,7 @@ void set_default_settings()
 	settings->setDefault("emergequeue_limit_diskonly", "128");
 	settings->setDefault("emergequeue_limit_generate", "128");
 	settings->setDefault("num_emerge_threads", "1");
-	settings->setDefault("secure.enable_security", "true");
+	settings->setDefault("secure.enable_security", "false");
```

Comment:
We should really just fix the games so we don't need this. I don't think it will be difficult, and it will make it much easier to play the games using official minetest releases.